leptonai/api/v1/types/job.py

```diff
@@ -39,6 +39,7 @@ class LeptonJobState(str, Enum):
     Completed = "Completed"
     Deleting = "Deleting"
     Restarting = "Restarting"
+    Archived = "Archived"
     Unknown = ""
 
```

Comment:
Similar to LeptonDeploymentState - would you mind overriding the __new__ method and try?